src/harness/reference_models/geo/census_tract.py

```diff
@@ -42,3 +45,149 @@ def GetCensusTract(self, fips_code):
           raise IOError('Error occurred in opening Census Tract File: %s' % census_tract_file.name)
     else:
       raise Exception("Census Tract data not found with FIPS Code: %s" % fips_code)
+
+  def __ExtractZipFiles(self, zip_filename=None):
```

Comment:
All those functions could be moved directly in your script

Reply:
Fixed   as per suggestion.